Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM postgres:10
+ARG DOCKER_POSTGRES_VERSION=10
+
+FROM postgres:$DOCKER_POSTGRES_VERSION
```

Comment:
It would be nice if you can also add variable for the base image name:
```Dockerfile
 ARG DOCKER_POSTGRES_IMAGE=postgres
```
because we would like to use the `camptocamp/postgres` as base image to be able to restore databases with postgis extension.

Reply:
Done!